pages/planetusermanual.md

```diff
@@ -218,6 +218,123 @@ Collections can be used by Nation Managers to categorize materials by topic for
 
 7. You will be directed to the same screen as the creation page. Change any of the information which should be updated and click “Save.” You can also delete or nest (sub-categorize) the Collection.
 
+## Courses Overview
+### What is a Course?
+A Course allows Leaders to walk Learners through specific Resources and exercises to learn new skills.  Leaders can create Courses by grouping and ordering Resources and allowing Learners to upload papers or take Quizzes.  
+
+### How do I create a Course?
+To create a Course, log in and start from the Home page.
+* Go to the *“Courses”* tab in the top menu and then select *“Add Course.”*
```

Comment:
The formatting of line 227, 231 and 265 should be the same according to the [original file](https://docs.google.com/document/d/1ikNCxVEJq22K8kcqkZIe0ibGDvL_1gMSl3Vk2F1_O8w/edit#heading=h.ois11f2go1wv). They look different on [rawgit](https://rawgit.com/sarah-lu102/sarah-lu102.github.io/manualcourses/#!pages/planetusermanual.md#Courses_Overview). not sure if it's work but maybe you can add an empty line between line 226 and 227. Also, there are some whitespace at the beginning of line 265